seed/static/seed/js/controllers/inventory_list_beta_controller.js

```diff
@@ -1136,6 +1169,33 @@ angular.module('BE.seed.controller.inventory_list_beta', [])
         }
       };
 
+      const operatorLookup = {
+        'ne': '!',
```

Comment:
I think this might need to be `!=`, see video below -- `!=` end up turning into a `contains !<value>`

https://user-images.githubusercontent.com/18518728/148571455-5684d3df-cc1e-4889-896f-4a8b52e22f10.mov

Reply:
Ah yeah, good catch